disnake/ext/commands/params.py

```diff
@@ -275,6 +276,45 @@ def __repr__(self) -> str:
         return f"{type(self).__name__}[{a}, {b}]"
 
 
+class StringMeta(type):
+    """Custom Generic implementation for String."""
+
+    def __getitem__(self, args: Tuple[Union[int, ellipsis], Union[int, ellipsis]]) -> Type[str]:
+        a, b = [None if x is ... else x for x in args]
+        return String.create(min_length=a, max_length=b)  # type: ignore # these are not ellipsis
```

Comment:
You wouldn't need the `# type: ignore` if you just put that over two lines

Reply:
wdym?

Reply:
Or actually look at the `__getitem__` in `RangeMeta`, it uses `is Ellipsis` and apparently that typechecks correctly

Reply:
It typechecks correctly because of the ``Any`` in the function definition. :clown: